crates/napi/scripts/constants.ts

```diff
@@ -77,8 +77,7 @@ export const languagesNodeTypesUrls = {
     "https://raw.githubusercontent.com/tree-sitter/tree-sitter-bash/refs/tags/{{TAG}}/src/node-types.json",
   [Lang.Yaml]:
     "https://raw.githubusercontent.com/tree-sitter-grammars/tree-sitter-yaml/refs/tags/{{TAG}}/src/node-types.json",
-  // The Lua tree-sitter repo does not have tags
   [Lang.Lua]:
-    "https://raw.githubusercontent.com/tjdevries/tree-sitter-lua/refs/heads/master/src/node-types.json",
+    "https://raw.githubusercontent.com/tree-sitter-grammars/tree-sitter-lua/refs/tags/{{TAG}}/src/node-types.json",
```

Comment:
@mohebifar note Google's first tree-sitter-lua repo isn't the one used. We are using https://crates.io/crates/tree-sitter-lua this